runtime/doc/builtin.txt

```diff
@@ -4785,8 +4787,8 @@ indexof({object}, {expr} [, {opt}])			*indexof()*
 
 		The optional argument {opt} is a Dict and supports the
 		following items:
-		    start	start evaluating {expr} at the item with index
-				{start} (may be negative for an item relative
+		    startidx	start evaluating {expr} at the item with index
+				staridx (may be negative for an item relative
```

Comment:
I think `staridx` should be replaced with `{startidx}`.

Reply:
I'll include the correction.